### PR TITLE
Create group when we create a workspace + backfill the past

### PR DESCRIPTION
**Pull request by PopDaph**
## Description

- create a `system` group when we create a workspace. 
- script to backfill a system` group for all existing workspaces. 

## Risk

Groups are not used yet so should not be too risky. 

## Deploy Plan

https://github.com/dust-tt/dust/pull/6474 must be merged first. 
Then deploy front. 
Then execute the backfill script.

### CODE REVIEW

File: front/admin/cli.ts [hunk 1]
```diff
@@ -43,6 +44,11 @@ const workspace = async (command: string, args: parseArgs.ParsedArgs) => {
         sId: generateLegacyModelSId(),
         name: args.name,
       });
+      await GroupResource.makeNew({
```
Comment (spolu) [inline front/admin/cli.ts:47]: We should also create the workspace group no?

**Review by spolu — APPROVED** [review]

File: front/admin/cli.ts (hunk 1 above)
Comment (PopDaph) [reply front/admin/cli.ts:47]: Ok added, only issue is now my dumb fill script will throw ugly errors if run multiple times but I'm okay with it if you're okay with it

Comment (spolu) [reply front/admin/cli.ts:47]: I'm ok with it

File: front/admin/cli.ts [hunk 2]
```diff
@@ -43,6 +44,16 @@ const workspace = async (command: string, args: parseArgs.ParsedArgs) => {
...
```
Comment (spolu) [inline front/admin/cli.ts:47]: Should be Promise.all on both?

**Review by spolu — APPROVED** [review]

**Pull request merged**